app/webpacker/components/Panel/pages/CreateNewComersPage/CompetitionsInput.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {
+  Button, Form, Header, HeaderSubheader,
+} from 'semantic-ui-react';
+import { IdWcaSearch } from '../../../SearchWidget/WcaSearch';
+import SEARCH_MODELS from '../../../SearchWidget/SearchModel';
+import { viewUrls } from '../../../../lib/requests/routes.js.erb';
+import useQueryParams from '../../../../lib/hooks/useQueryParams';
+import useInputState from '../../../../lib/hooks/useInputState';
+
+export default function CompetitionsInput() {
+  const [queryParams] = useQueryParams();
+  const competitionIdsFromQuery = queryParams?.competition_ids?.split(',');
+  const [competitionIds, setCompetitionIds] = useInputState(competitionIdsFromQuery || []);
+
+  return (
+    <Form>
+      <IdWcaSearch
+        model={SEARCH_MODELS.competition}
+        multiple
+        value={competitionIds}
+        onChange={setCompetitionIds}
+        label="Competition(s)"
+      />
+      <Header as="h4">
+        <HeaderSubheader>
+          Leave blank to check for all competitions
+        </HeaderSubheader>
+      </Header>
```

Comment:
Creating a header only for showing a subheader (with no "main" header) smells like bad code to me. What is your goal here, and why is a simple text paragraph not good enough?

Reply:
I wanted to show it as 'hint' like thing, so I followed your suggestion in another PR ([LINK](https://github.com/thewca/worldcubeassociation.org/pull/10652#discussion_r1922373248)). Did I implement this wrongly?